src/logger.ts

```diff
@@ -1,37 +1,45 @@
 import { printHtml } from "kolmafia";
 
 const defaultHandlers = {
-  info: (message: string) => printHtml(`<b>[Libram]</b> ${message}`),
-  warning: (message: string) =>
+  info: (message: string): unknown => printHtml(`<b>[Libram]</b> ${message}`),
+  warning: (message: string): unknown =>
     printHtml(
       `<span style="background: orange; color: white;"><b>[Libram]</b> ${message}</span>`
     ),
-  error: (error: string | Error) =>
+  error: (error: string | Error): unknown =>
     printHtml(
       `<span style="background: red; color: white;"><b>[Libram]</b> ${error.toString()}</span>`
     ),
+  debug: (message: string): unknown => {
+    if (Logger.debugMode) {
+      printHtml(
+        `<span style="background: red; color: white;"><b>[Libram Debug]</b> ${message}</span>`
+      );
+      return;
+    }
+  },
 };
 
 export type LogLevel = keyof typeof defaultHandlers;
+type LogFunction<T extends LogLevel> = typeof defaultHandlers[T];
 
 class Logger {
   handlers = defaultHandlers;
+  private static debug = false;
 
-  setHandler(
-    level: "error",
-    callback: (message: string | Error) => unknown
-  ): void;
-  setHandler(
-    level: "warning" | "info",
-    callback: (message: string) => unknown
-  ): void;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  setHandler(level: LogLevel, callback: (message: any) => unknown): void {
+  static get debugMode(): boolean {
+    return Logger.debug;
+  }
+  static set debugActive(debug: boolean) {
```

Comment:
this should probably just set a level instead of having specific stuff for debug

Reply:
none < error < warning < message < debug, right? So if debug is on we want to show everything, if 'message' is on we want to show warnings and errors too, etc etc?